tensorrt_llm/_torch/distributed/communicator.py

```
@@ -341,9 +342,30 @@ class MPIDist(Distributed):
 
     def __init__(self, mapping: Mapping):
         super().__init__(mapping)
+        self.create_cp_comm()
+        # Repurpose CP ranks to TP for Helix so that the right comms are created.
+        mapping_with_helix = None
+        if self.mapping.cp_size > 1:
+            print(f"[MPIDist::__init__] Repurposing CP ranks to TP for Helix.")
```

Ruff (0.14.5)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)